### DISCUSSION TITLE
LCD Progress bar not updating

### DISCUSSION THREAD

**Discussion by wbvczar** [Jyers/Marlin#195] [Q&A] [answered]
I have flashed the Manual_Mesh_3x3 pre-compiled firmware (v 1.1.1), and I have added the M75 command in the "GCode Scripts" of OctoPrint in the "Before Print Job Starts" section.  But when I print, I get the progress bar on the LCD screen, but it never moves off of 0%.  It updates percentage and the layers layers completed below elapsed time, but the actual progress bar doesn't update.

Should the progress bar be updating?

**Comment by Jyers** [comment, marked as answer]
You need an OctoPrint plugin to send M73 commands, I use "M73 Progress"
